gnome-initial-setup/gis-driver.c

```diff
@@ -170,7 +173,7 @@ udisks_client_new_cb (GObject      *source,
 
       g_list_free_full (blocks, g_object_unref);
     }
-  else
+  else if (!g_cancellable_is_cancelled (priv->cancellable))
```

Comment:
Optional: you could check here if the error matches the cancelled code instead of looking directly at the cancellable state.

Reply:
Ok